CONTRIBUTING.md

```diff
@@ -297,6 +302,8 @@ $ ./cli exec
 root@9feae5e5e001:/src/conjur-server#
 ```
 
+###### Run Cukes with Open ID Connect (OIDC) Compatible Environment
```

Comment:
Please update this too

###### Spin up Open ID Connect (OIDC) Compatible Environment for testing